tests/conftest.py

```diff
@@ -388,6 +391,62 @@ def running_clients(hlwm, running_clients_num):
     """
     return hlwm.create_clients(running_clients_num)
 
+@pytest.fixture()
+def x11():
+    from Xlib import X, display, Xutil
+    class X11:
+        def __init__(self):
+            self.display = display.Display()
+            self.screen = self.display.screen()
+            self.root = self.screen.root
+
+        def window(self, winid_string):
+            """return python-xlib window wrapper for a string window id"""
+            winid_int = int(winid_string, 0)
+            return self.display.create_resource_object('window', winid_int)
+
+        def winid_str(self, window_handle):
+            return hex(window_handle.id)
+
+        def make_window_urgent(self, window):
+            """make window urgent"""
+            window.set_wm_hints( flags = Xutil.UrgencyHint )
+            self.display.sync()
+
+        def is_window_urgent(self, window):
+            """check urgency of a given window handle"""
+            hints = window.get_wm_hints()
+            if hints is None:
+                return False
+            return bool(hints.flags & Xutil.UrgencyHint)
+
+        def create_client(self, urgent=False):
```

Comment:
Will this end up replacing `HlwmBridge.create_client()`? I'm just wondering if we could avoid confusion due to the same name, but OTOH, the name is fine in itself.

Reply:
Yes, I'd vote for using x11.create_client() in the long run (or actually: using it in hlwm.create_client()). I think this will also simplify the keymask-tests a lot.

Reply:
Nice, looking forward to it.